libshpool/src/daemon/server.rs

```diff
@@ -480,6 +482,7 @@ impl Server {
     fn handle_kill(&self, mut stream: UnixStream, request: KillRequest) -> anyhow::Result<()> {
         let mut not_found_sessions = vec![];
         {
+            let _s = span!(Level::INFO, "lock(shells)").entered();
```

Comment:
nit: we have a few duplicate strings here which might be hard to search, should we rename to something more specific like you did for other lines?

maybe `handle_kill_lock(shells)` here and `handle_list_lock(shells)` or similar?

Reply:
These will be inside the scopes for the functions they appear in, so they will be differentiated that way. The way this will look in the logs is `handle_kill:lock(shells): new`

Reply:
oh i didn't know that, neat yeah this makes sense, sg!